THIRD_PARTY_LICENSES.txt

```diff
@@ -10551,6 +10562,8 @@ github.com/peterbourgon/diskv
 github.com/posener/complete
 github.com/robfig/cron
 github.com/rs/cors
+github.com/rs/xid
```

Comment:
what is this? you cannot add a dependency without approval

Reply:
I believe this might be an indirect dep @markxnelson  - if you look at go.mod it's not listed there

Reply:
if it is indirect it should not be here :)